chcemvediet/apps/anonymization/cron.py

```diff
@@ -25,6 +33,47 @@ def attachment_normalization():
             content_type=content_types.PDF_CONTENT_TYPE
         )
         cron_logger.info(u'Normalized attachment: {}'.format(attachment))
+
+    elif attachment.content_type in content_types.LIBREOFFICE_CONTENT_TYPES:
+        directory_name = tempfile.mkdtemp(dir=u'.')
+        file_copy = os.path.join(directory_name, u'file' + guess_extension(attachment.content_type))
```

Comment:
`file_copy` nie je dobry nazov, kedze premenna neobsahuje kopiu suboru, ale cestu ku suboru. Nazvi to `file_path` alebo tak nejak.

Reply:
:ok: